gradebookng/tool/src/webapp/scripts/gradebook-grade-summary.js

```diff
@@ -266,26 +266,30 @@ GradebookGradeSummary.prototype.setupStudentView = function() {
 
 
 GradebookGradeSummary.prototype._print = function(headerHTML, contentHTML, $container) {
-  $("#summaryForPrint").remove();
+  let $iframe = $("#summaryForPrint");
+  if ($iframe.length === 0) {
+    $iframe = $("<iframe id='summaryForPrint'>").hide();
+    $container.append($iframe);
+    $iframe.attr("src", "about:blank");
+    $iframe.one("load", () => setTimeout(() => this._print(headerHTML, contentHTML, $container), 500));
+    return;
+  }
 
-  var $iframe = $("<iframe id='summaryForPrint'>").hide();
-  $iframe.one("load", function() {
-    var $head = $iframe.contents().find("head");
-    $(document.head).find("link").each(function() {
+  $container.find("#summaryForPrint").attr("src", "about:blank");
```

Comment:
summaryForPrint should be ID unique:

$("#summaryForPrint").attr("src", "about:blank");